project/src/World/DockingPoint/DockingPoint.gd

```diff
@@ -24,6 +24,8 @@ onready var remote_transform: RemoteTransform2D = $RemoteRig/RemoteTransform2D
 onready var ref_to := weakref(self)
 onready var tween := $TweenAura
 onready var dock_aura := $DockingAura
+onready var player_rotation_transform = $Sprite/PlayerRotationRig/PlayerRotationRemoteTransform
+onready var player_rotation_transform_rig = $Sprite/PlayerRotationRig
 
 
 func _ready() -> void:
```

Comment:
Because the Sprite's scale is not guaranteed to be (1,1), and the remote transform rig is a child of the sprite, its scale will change too.

To simulate it being toplevel without being toplevel (as we want to maintain rotation and position), you should set its scale to the inverse of the sprite's scale in _ready. I.E.:
```gdscript
player_rotation_transform_rig.scale = Vector2.ONE / sprite.scale
```

Reply:
Thank you! Did this in bfeb0e7a7b8052e2fe297833efa58838bcc84abd. I don't quite understand why setting it in ready is sufficient. I would've thought I'd need to call this every time I dock.

Reply:
We don't have any mechanics for changing the scale of a docking point, so that scale is constant from the start of its lifespan to its end. That's why we don't need to set it again outside of ready. The sprite.scale it refers to is the sprite of the docking point.